cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -109,13 +112,22 @@ def test_snapshot_crud(small_test_vm, provider):
         casecomponent: Infra
         initialEstimate: 1/6h
     """
+    result = LogValidator(
+        "/var/www/miq/vmdb/log/evm.log",
+        failure_patterns=[r".*ERROR.*"],
+    )
+    result.start_monitoring()
     # has_name is false if testing RHEVMProvider
     snapshot = new_snapshot(small_test_vm, has_name=(not provider.one_of(RHEVMProvider)))
     snapshot.create()
     # check for the size as "read" check
     if provider.appliance.version >= "5.11" and provider.one_of(RHEVMProvider):
         assert snapshot.size
     snapshot.delete()
+    wait_for(lambda: not snapshot.exists)
```

Comment:
Q: any `timeout` to assign here?

Reply:
@john-dupuy default timeout is 2 mins.. Do you think that it would be better to increase the timeout? I don't know what would be reasonable one for snapshot removal.

Reply:
@psimovec I just realized that the Snapshot `delete` method will also include the `wait_for`, so I would consider this one redundant. 

https://github.com/ManageIQ/integration_tests/blob/5dc69946af7d4f25ba1f0bc5a1a9a2e4978fce67/cfme/infrastructure/virtual_machines.py#L756-L775